### DISCUSSION TITLE
1.8.9 support

### DISCUSSION THREAD

**Discussion by dodi2020** [gorilla-devs/ferium#268] [Ideas]
1.8.9 support would be helpful since lots of people like 1.8.9 for servers like hypixel and would probably enjoy auto updating mods

**Comment by theRookieCoder** [comment]
Every Minecraft version, including snapshots and even pre-alpha, is supported (for that matter, you can literally use any string). Till now these had to be manually entered in using the CLI or by editing the config file, but now I have added support to pick any minecraft version in 2713a2e5f50267c35e3f8e95118bb4b6d2670f1c [`v4.3.4`](https://github.com/gorilla-devs/ferium/releases/tag/v4.3.4)

**Reply by dodi2020** [reply]
Well then, for some reason i dont have more options when making a new profile..
Its only 1.13.2+ i think

**Reply by theRookieCoder** [reply]
You have to update the program to the latest version

**Reply by dodi2020** [reply]
ik, i downloaded the latest version(4.3.4) and replaced the old one..

**Reply by theRookieCoder** [reply]
Does `ferium --version` actually return `ferium 4.3.4`? The picker works fine for me

**Reply by dodi2020** [reply]
oh what?! it says 4.1.10..
even tho i changed the ferium.exe to the new one
:/ what do i do

**Reply by theRookieCoder** [reply]
Probably could have installed from multiple sources, or your PATH order is messed up.

**Reply by dodi2020** [reply]
hm.. i tried it again and it doesnt work..
